tests/tests.py

```diff
@@ -88,24 +98,21 @@ def offline_test(model = "mistralai/Mistral-7B-Instruct-v0.2") -> pd.DataFrame:
     pattern = "Server started at"
     run_command("lmcache_server localhost 65431", stdout_log, stderr_log, detach=True)
     time.sleep(10)
-    os.environ['LMCACHE_CONFIG_FILE'] = "/local/shaotingf/lmcache1/lmcache-tests/configs/example_offline.yaml"
+    os.environ['LMCACHE_CONFIG_FILE'] = "/local/shaotingf/lmcache1/lmcache-tests/configs/example.yaml"
```

Comment:
Can we avoid using hard-coded paths here? Is it okay to just pass "./configs/example.yaml" here? (which requires to run the test in the project root folder)